catalog-info.yaml

```diff
@@ -108,6 +108,45 @@ spec:
         everyone:
           access_level: BUILD_AND_READ
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elastic-agent-testing-bk
```

Comment:
Why are we adding a new BK pipeline and not re-using the existing one?

If we need a new one, I propose splitting this into a dedicated PR so that we can test the actual pipeline before merging the PR.

Reply:
There is an agreement stating that we'll have a grace period when we have both: GCP and BK-based tests to ensure we run the same tests.

Reply:
Created one : https://github.com/elastic/elastic-agent/pull/5717